packages/calypso-e2e/src/lib/pages/media-page.ts

```diff
@@ -90,16 +91,32 @@ export class MediaPage {
 	}
 
 	/**
-	 * Given that a gallery item is selected, enter the edit screen.
+	 * Launches the edit item modal where file attributes can be modified.
 	 *
-	 * @returns {Promise<void>} No return value.
+	 * This method expects at least one gallery element supporting editing
+	 * to be selected. If no images are selected, this method will throw.
+	 *
+	 * @throws {Error} If no gallery items are selected.
 	 */
-	async editImage(): Promise< void > {
-		await this.waitUntilLoaded();
+	async editItem(): Promise< void > {
```

Comment:
Optional suggestion: what do you think of naming this `editSelectedImage`? In reading through the scripts using this, I get turned around with the `editImage` method having such a similar name, and I keep expecting to see this function take an argument (but it doesn't, because it expects one to be already expected)

Reply:
Sure thing, will look at renaming the `editItem` and `editImage` methods.

Reply:
Fixed in f243d0b7576c7e66e874edc48703cf17d2d288e1.